src/stories/colorpalette.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Divider, Typography } from '@material-ui/core';
+
+import { StyleTyping, SampleStyleTyping, defaultStyles } from './BaseComponents/Utils';
+
+export default {
+  title: 'Color Palette',
+};
+
+const styles: StyleTyping = {
+  ...defaultStyles,
+  example: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    margin: '1rem 0',
+  },
+};
+
+const sampleStyles: SampleStyleTyping = {
+  colorSample: {
+    unknown: {
+      backgroundColor: '#cccccc',
```

Comment:
Don't hardcode this, if we are standardizing the color palette, we should add a class to reference the colors.